### ISSUE TITLE
Native function request

### ISSUE THREAD

**Issue by suzuki-shunsuke** [lintnet/lintnet#703]
✅ Answered by [suzuki-shunsuke](https://github.com/suzuki-shunsuke) <a href="#DC_kwDOK40UVM4Ak1_L">Answer</a>

<img width="32" alt="image" src="https://avatars.githubusercontent.com/u/13323303?u=afedf0091bfd70a6a79c55f6aca781c94cb862f7&v=4"> [suzuki-shunsuke](https://github.com/suzuki-shunsuke) 2024-06-03T08:50:03Z ⬆️ 1 

[Original Discussion](https://github.com/lintnet/lintnet/discussions/472)
Category: :ghost: General

lintnet supports native functions.

https://lintnet.github.io/docs/lint-rule/native-function/

If you want a new function, please post a new comment rather than replying to a thread.
Before posting a comment, please check if the function has already been supported or requested.

Please vote to the function you want rather than posting a comment like `+1`.

Basically, we don't accept a function accessing filesystem (e.g. [os.WriteFile](https://pkg.go.dev/os#WriteFile)) and network (e.g. [net/http#Get](https://pkg.go.dev/net/http#Get)) and executing external functions (e.g. [os/exec/#Cmd.Run](https://pkg.go.dev/os/exec#Cmd.Run)).

## Comment Format

```md
# <function name>

- <Reference URL>

<Note>
```

If the function has the Go implementation, please tell us the link to Go Doc.

e.g.

```md
# strings.Contains

- https://pkg.go.dev/strings#Contains
```

_[This issue was created by ghd2i](https://github.com/suzuki-shunsuke/ghd2i)_

**Comment by suzuki-shunsuke** [comment]
# ✅ Mark as Answer


[Original Comment](https://github.com/lintnet/lintnet/discussions/472#discussioncomment-9658315) | _[Created by ghd2i](https://github.com/suzuki-shunsuke/ghd2i)_
<img width="32" alt="image" src="https://avatars.githubusercontent.com/u/13323303?u=afedf0091bfd70a6a79c55f6aca781c94cb862f7&v=4"> [suzuki-shunsuke](https://github.com/suzuki-shunsuke) <a href="#DC_kwDOK40UVM4Ak1_L" id="DC_kwDOK40UVM4Ak1_L">2024-06-04T05:07:00Z</a> ⬆️ 1 

Use the issue template instead. https://github.com/lintnet/lintnet/issues/new?assignees=&labels=enhancement%2Cnew-native-function&projects=&template=new-native-function-request.yml
